KomodoPlatform/komodo-wallet-desktop
# Fix dex overflow

Closes: https://github.com/KomodoPlatform/komodo-wallet-desktop/issues/2345

This PR also sets min height to 720px (down from 800px), which may introduce some overflows at the bottom of the app. Margins and components have been reduced to accommodate this, though some places may still need work. 

To test:
- Go to dex pro view
- Interact with all elements and confirm no overflows are seen.
- Check all pop up windows in app to confirm working and visually ok (including from outside dex page. E.g. adding coins, viewing orders etc)
- Check any columns where hidden components (e.g. error messages) may result in elements being pushed outside the bottom of the app window.
